### ISSUE TITLE
Maven3-Artifactory Integration not persisting Target repositories v3.8.1

### ISSUE THREAD

**Issue by trevormarshall** [jfrog/jenkins-artifactory-plugin#346]
**Describe the bug**
After we save the 'Target releases repository' and 'Target snapshots repository' in 'Maven3-Artifactory Integration', they are not persisted to `config.xml`.
Builds fail with:
```
[main] ERROR org.apache.maven.cli.MavenCli - Target repository cannot be empty
java.lang.IllegalArgumentException: Target repository cannot be empty
```

**To Reproduce**
- We create either a freestyle job or a pipeline from seed 
- select 'Maven3-Artifactory Integration'
- choose our Artifactory server (which tests OK in the general configuration -> JFrog config)
- click "Refresh Repositories"
- select valid 'Target releases repository' and 'Target snapshots repository' 
- Save
- refresh the config page or check `config.xml`, Target repos are not present.

**Expected behaviour**
Target repos are saved in `config.xml` and can be viewed in the plan configuration, build uses these repositories.

**Versions**
- Jenkins Artifactory plugin version: 3.8.1 and also tested 3.8.0
- Jenkins operating system: 2.249.1 Ubuntu
- Artifactory Version: SaaS, plugin test says "Found Artifactory 6.18.1"

**Additional context**
- No errors or warnings are seen in `jenkins.log` when we save the plan config.
- Originally we had a proxy configured in "Manage Plugins -> Advanced", but removed that completely. 
- Tried downgrading plugin from 3.8.1 to 3.8.0 but the issue persists.

**Comment by yahavi** [comment]
Thanks for reporting this issue, @trevormarshall.

We fix it in #341. The fix will be included in the next release.

Here is a snapshot with the fix:
https://oss.jfrog.org/oss-snapshot-local/org/jenkins-ci/plugins/artifactory/3.8.x-SNAPSHOT/artifactory-3.8.x-20200921.093853-4.hpi

We'd appreciate your feedback for that!

**Comment by trevormarshall** [comment]
@yahavi I installed the snapshot plugin, configured a test plan, saved, and indeed it appears to have resolved this issue for us. Thanks for the speedy notification, greatly appreciated.
